### PR TITLE
fix: grid item skeleton cards size issue

### LINKED ISSUE #9377: Items grid skelton cards size doesn't  match normal cards size
### What happened?

CleanShot 2024-02-12 at 15 33 45

### PR DESCRIPTION
**Pull request by floyd-li**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #9377
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=13rv1SWoLg9Gb3tmvHPZxb7JbVy51BtMziX7k9WQGSJ7Kp3A)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/koda_eco)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

https://github.com/kodadot/nft-gallery/assets/16473062/40fc9e4b-e645-4f64-b6d9-a886fc9fa13d

### CODE REVIEW

File: components/shared/nftCard/NftCardSkeleton.vue [hunk 1]
```diff
@@ -24,6 +27,7 @@ withDefaults(
   defineProps<{
     hideMediaInfo?: boolean
     slimMediaInfo?: boolean
+    minimal?: boolean
```
Comment (hassnian) [inline components/shared/nftCard/NftCardSkeleton.vue:30]: combine into one , `variant` prop

File: components/shared/nftCard/NftCardSkeleton.vue [hunk 2]
```diff
@@ -8,9 +8,12 @@
     <div
       v-if="!hideMediaInfo"
       class="nft-media-info"
-      :class="{ 'nft-media-info__slim': slimMediaInfo }">
+      :class="{
+        'nft-media-info__slim': slimMediaInfo,
+        'nft-media-info__minimal': minimal,
+      }">
```
Comment (hassnian) [inline components/shared/nftCard/NftCardSkeleton.vue:14]: do like this 

NftCard.vue

CleanShot 2024-04-10 at 12 36 14@2x

**Review by hassnian — CHANGES_REQUESTED** [review]

File: components/items/ItemsGrid/ItemsGrid.vue [hunk 3]
```diff
@@ -234,4 +236,13 @@ useIntersectionObserver(target, async ([{ isIntersecting }]) => {
...
 })
+
+const getSkeletonVariant = (slotProps) => {
+  if (slotProps.isMobileVariant || slotProps.grid === 'small') {
+    return 'minimal'
+  }
+  if (props.collectionPopoverHide) {
+    return 'slim'
+  }
+}
```
Comment (daiagi) [inline components/items/ItemsGrid/ItemsGrid.vue:247]: missing a default return value

**Comment by prury** [comment]
@floyd-li please resolve conversations

**Comment by prury** [comment]
there is a super small skeleton shifting when you browse profile, no need to fix if its too complicated

**Review by daiagi — APPROVED** [review]

**Review by hassnian — APPROVED** [review]

**Comment by prury** [comment]
@daiagi please check if conversations are good to go

**Comment by daiagi** [comment]
@prury 

all good. may continue

**Comment by prury** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 8.26 USD/DOT ~ 3.632 $DOT
🧗 [13rv1SWoLg9Gb3tmvHPZxb7JbVy51BtMziX7k9WQGSJ7Kp3A](https://polkadot.subscan.io/account/13rv1SWoLg9Gb3tmvHPZxb7JbVy51BtMziX7k9WQGSJ7Kp3A)
🔗 [0xd2e1da10fec9d57ba3e7b382bd2e8d294e0a3d0abb08412bef188ea1707dd51b](https://polkadot.subscan.io/extrinsic/0xd2e1da10fec9d57ba3e7b382bd2e8d294e0a3d0abb08412bef188ea1707dd51b)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**